### DISCUSSION TITLE
strange error with databinding custom synthesizers.

### DISCUSSION THREAD

**Discussion by Aaron1178** [livewire/livewire#7613] [1. Help]
Hello,

Edit: I am using the latest Laravel / Livewire / Alpinejs versions.

I have a custom synthesizer that is working as expected. I am however noticing a strange bug when using wire:model.live on text inputs. 

When I start typing on a text input with wire:model.live, it triggers an update as soon as i start typing, resulting in a few (possibly 1) character being deleted. I can after that, proceed with typing normally, and everything works as expected. 

This problem doesn't occur if I change the input to wire:model.lazy. 

I've installed the livewire dev tools extension for chrome, and also the barryvfh debug toolbar, but for the life of me, I cannot find why this is occurring.

This problem also doesn't occur if the field is data bound to a primitive type on my livewire component, which is why I've classified it in the Synthesizer category.

my synth code is below:
`<?php

namespace App\Classes\QuoteEditor;

use Livewire\Mechanisms\HandleComponents\Synthesizers\Synth;

class QuoteSynth extends Synth
{
    public static $key = 'quote';

    public static function match($target)
    {
        return $target instanceof Quote;
    }

    public function dehydrate($target)
    {
        return [[
            'id' => $target->id,
            'reference' => $target->reference,
            'client' => $target->client,
            'worksAddress' => $target->worksAddress,
            'notes' => $target->notes,
            'date' => $target->date,
            'duration' => $target->duration,
            'estimator' => $target->estimator,
            'materials' => $target->materials,
            'mCurrentlyEditing' => $target->mCurrentlyEditing,
            'labour' => $target->labour,
            'lCurrentlyEditing' => $target->lCurrentlyEditing,
            'misc' => $target->misc,
            'miCurrentlyEditing' => $target->miCurrentlyEditing,
            'attachments' => $target->attachments,
            'netTotal' => $target->netTotal,
            'tax' => $target->tax,
            'total' => $target->total,
            'toggled' => $target->toggled,
            'dirty' => $target->isDirty(),
        ], []];
    }

    public function hydrate($value)
    {
        $instance = new Quote();

        $instance->id = $value['id'];
        $instance->reference = $value['reference'];
        $instance->client = $value['client'];
        $instance->worksAddress = $value['worksAddress'];
        $instance->notes = $value['notes'];
        $instance->date = $value['date'];
        $instance->duration = $value['duration'];
        $instance->estimator = $value['estimator'];
        $instance->materials = $value['materials'];
        $instance->mCurrentlyEditing = $value['mCurrentlyEditing'];
        $instance->labour = $value['labour'];
        $instance->lCurrentlyEditing = $value['lCurrentlyEditing'];
        $instance->misc = $value['misc'];
        $instance->miCurrentlyEditing = $value['miCurrentlyEditing'];
        $instance->attachments = $value['attachments'];
        $instance->netTotal = $value['netTotal'];
        $instance->tax = $value['tax'];
        $instance->total = $value['total'];
        $instance->toggled = $value['toggled'];
        $instance->dirty = $value['dirty'];

        return $instance;
    }

    public function get(&$target, $key)
    {
        if (is_array($target))
        {
            return $target[$key] ?? null;
        }

        return $target->{$key};
    }

    public function set(&$target, $key, $value)
    {
        $target->{$key} = $value;
    }
}
`

Thanks for your help and thanks Livewire team for the awesome bit of kit :)

**Comment by akamasAT** [comment]
i noticed this when my internet connection is slow. try adding debounce
https://livewire.laravel.com/docs/forms#debouncing-input

**Reply by Aaron1178** [reply]
non debouncing would not be the issue as I am developing on a local dev environment.
